src/Microsoft.AspNetCore.Http/Internal/DefaultConnectionInfo.cs

```diff
@@ -20,25 +20,21 @@ public class DefaultConnectionInfo : ConnectionInfo
 
         public DefaultConnectionInfo(IFeatureCollection features)
         {
-            Initialize(features);
```

Comment:
Yes, it is. I thought it made sense to clean it up when touching the file anyway. I can strip the last commit if you want...